duden/word.py

```diff
@@ -357,3 +372,15 @@ def phonetic(self):
             return ipa.get_text()
 
         return None
+
+    @property
+    def alternative_spellings(self):
+        """
+        Returns alternate spellings
+        """
+        alternative_spellings = self.soup.find_all('span', {"class": "lemma__alt-spelling"})
+        if alternative_spellings is not None:
+            for spelling in alternative_spellings:
+                return spelling.get_text()
+
+        return None
```

Comment:
If the property is called spellings, then I think it should return a list of spellings instead of a single spelling. I don't know about any examples (and would like to know, please let me know if you find any), but there may be words with several alternate spellings. And for them it might be useful to return all of them - not just the first one. If we are to return a single element, it should be called `alternate_spelling`.

What do you think?
```suggestion
        if alternative_spellings is None:
            return None

        return [spelling.get_text() for spelling in alternate_spellings]
```